recipe/install-libgomp.sh

```diff
@@ -17,6 +17,7 @@ mkdir -p ${PREFIX}/${CHOST}
 cp -r ${tmp_dir}/sysroot ${PREFIX}/${CHOST}/sysroot
 mkdir -p ${PREFIX}/lib
 cp ${tmp_dir}/libgomp.so.${libgomp_ver} ${PREFIX}/lib/libgomp.so.${libgomp_ver}
+ln -s ${PREFIX}/lib/libgomp.so.${libgomp_ver} ${PREFIX}/lib/libgomp.so
```

Comment:
Ohhh you moved it here. Why?

Reply:
That link is the only link that would clobber installing two compilers in the same environment.

Reply:
Right because it is not in the sysroot. But why did we move it to there in the first place?

Looks like the last commit was this PR: https://github.com/conda-forge/ctng-compilers-feedstock/pull/13

Reply:
I am still wondering if you have thoughts on this thread.

Reply:
I can't remember why we did, but the link `ln -s ${PREFIX}/libgomp.so ${PREFIX}/${CHOST}/sysroot/lib/libgomp.so` is wrong because it is linking a `native` library into a cross compiling dir which is wrong because the two architectures might be different.